sdp.go

```diff
@@ -12,23 +13,52 @@ import (
 	"github.com/pion/sdp/v2"
 )
 
+const (
+	// TODO(sgotti) add these to pion/sdp
+	extMapValueSDESMid         = 4
+	extMapValueSDESRTPStreamID = 5
```

Comment:
Is this the value that Chrome just happens to use for these extensions or is fixed in spec?